user_management/doc/User_management.md

```diff
@@ -265,6 +265,8 @@ will return public user information
 
 ### Parameters
 
+Authorization: {access_token}
+
```

Comment:
Please note that this is not a parameter in the request, but rather a header. These are 2 different things.

Reply:
[f95af4a](https://github.com/tdameros/42-transcendence/pull/217/commits/f95af4abf5d13d6791009754c6c0d786933164a7)